src/custom/pages/App/index.tsx

```diff
@@ -89,12 +89,21 @@ function createRedirectExternal(url: string) {
 
 const Loading = <LoadingWrapper>Loading...</LoadingWrapper>
 
+const getShowBannerState = (key: string) => {
+  const value = localStorage.getItem(key)
+  if (value === 'false') {
+    return false
+  }
+
+  return true
+}
+
 export default function App() {
   return (
```

Comment:
yes on the line above add a useMemo hook to get this using the fn you already made
```js
const isVisible = useMemo(() => getShowBannerState(BANNER_KEY), []) // empty array only calls on mount
```

Reply:
Refactored now, stored the localStorage as a CONST on top of the file.